automation_tools/__init__.py

```diff
@@ -418,6 +418,11 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
+def setup_scap_client():
+    """Task to setup puppet-foreman_scap_client."""
+    run('yum -y puppet-foreman_scap_client', warn_only=True)
```

Comment:
I think before directly running installation. we can make sure that scap_client is available for install from yum. By doing yum list.